lib/system/plugin-vue-loader.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Single File Vue Component Loader.
+ */
+/* global System, URL, fetch */
```

Comment:
these globals could be defined in an .eslintrc file in this subdir that specifies an appropriate environment or globals like this: https://github.com/digitalbazaar/bedrock-ledger-agent/blob/master/test/mocha/.eslintrc#L5